docs/products/opensearch/howto/connect-with-python.rst

```diff
@@ -0,0 +1,45 @@
+Connect to OpenSearch cluster with Python
+=========================================
+
+You can interact with your cluster with the help of the `Python OpenSearch client <https://github.com/opensearch-project/opensearch-py>`_. 
+It provides an easy syntax to send commands to your OpenSearch cluster. Follow its ``README`` file for installation instructions.
+
+To connect with your cluster, you need the **Service URI** of your OpenSearch cluster. You can find your cluster URI in the `**Overview** <https://console.aiven.io/project/dev-sandbox/services>`_ page. Alternatively, you can find it in Aiven command line interface `service command <https://developer.aiven.io/docs/tools/cli/service.html#avn-service-get>`_. ``service_uri`` contains credentials, therefore should be treated with care.
```

Comment:
sorry to be picky :D the phrase "you can find it in Aiven command line interface" seems a bit off, since you need to use the cli to get it. maybe something around " Alternatively, you can retrieve it by running `avn-service-get command<https://developer.aiven.io/docs/tools/cli/service.html#avn-service-get>`_" in Aiven command line interface"

Reply:
also, since https://developer.aiven.io/docs/tools/cli/service.html#avn-service-get is same website, better not to refer to https://developer.aiven.io, but to use an internal link. We use  ":doc:" and ":ref:" for relative paths.
P.S. I know this is probably copied from some of my stuff where I made same mistake in the past :D